Add blogging guide, communications plan, index tidying (#539)
publiccodenet/about: activities/communication/communications-plan.md

---
type: resource
---

# Communications plan

The Foundation for Public Code relies on communications to [achieve its mission](../../organization/mission.md).

## Principles

Our communications need to reflect our core values: open, quality, trust, community, helpful. They must also be non-partisan, in line with our [founding principles](../../organization/mission.md#founding-principles).

Our messages range from neutral in tone to positive and inspirational. Our community won’t be sustainable if we attract people motivated by fear or anger.

Our communications are in plain English, but we link to highly relevant pages in other languages, for example about our members’ work.

> Our communications should show to showcase the Foundation for Public Code as:

Reviewer: ```suggestion
Our communications should showcase the Foundation for Public Code as:
```

* active, dynamic, getting things done
* smart enough to be trusted with the importance and complexity of its work
* plugged into and supportive of the network of visionaries and activists who also want this future
* open and transparent, both about its intentions and operations as well as learnings, failures and successes

We believe it’s more important to talk concretely about what we’ve already done than what we hope to do.

## Objectives

> Our communications should help us achieve [our current priorities](../../organization/mission.md#priorities).

Reviewer 2: The heading is named 'Current priorities'

```suggestion
Our communications should help us achieve [our current priorities](../../organization/mission.md#current-priorities).
```

## Audience

Our audience is divided into:

* current Foundation contributors
* future Foundation contributors

> * influencers

Reviewer: I find this very focussed on "us". Is our audience not also (people working in/for) public organizations on public code transformation, or OSS? How can we leverage the broader public code community to achieve our current priorities? This doesn't all have to be answered in this PR, but I think something to think about.

Author: Optimistically, all the people you list are future Foundation contributors, in the broadest sense of our community.

This list used to be [all of these people](https://about.publiccode.net/activities/value-and-impact/user-mapping/), but was cut to this to make it consistent with the very specific way we've defined 'user' (which hasn't been updated in value and impact mapping).

### Foundation contributors

We believe [these are potential contributors to the Foundation for Public Code](https://about.publiccode.net/activities/value-and-impact/user-mapping/).

### Influencers

> These are domain celebrities in open source, government digital transformation and civic tech.

Reviewer: As in individual humans, or whole organizations like FSFE?

Reviewer 3: Why not both?

Author: It's a mix of both.

Author: Updated!

## Unique selling proposition

> We are the only organization focused on open source software for public organizations that:

Reviewer 2: This list would make more sense if it was clear that not any of these but all of these together make us what we are, as all of them are done by others.

> * is non-profit and member owned (unlike commercial open source govtech companies)

Reviewer 2: This needs a qualifier 'with membership only open to public organizations' if this needs to be true on it's own.

Author: done.

> * offers codebase stewardship for different projects (how are we different from the Commons Conservancy and the businesses they can put you in touch with?)

Reviewer 2: I don't completely know what this means.

Reviewer: I understand this as us offering codebase stewardship to a number of public code projects, unlike existing public code foundations taht focus on one codebase e.g. Decidim, Consul foundations. 

I think the sentence in the brackets is confusing.

* offers productization and marketing of the codebases (unlike Commons Conservancy)
* actually provides guidance and support to help public orgs make their code open (unlike Free Software Foundation)
* isn’t tied to the EU project lifecycle (unlike SCORE)
* has a reproducible process suitable not just for cities, but all public organizations (unlike SCORE)
* is international (unlike OS2, Cloud 4 Cities, VNG-Realisatie, Adullact)

We are also actively building our ecosystem by:

> * connecting members with like-minded parties (potential members, partners, affiliates, structural funders)

Reviewer: ```suggestion
* connecting members with like-minded parties (other members, affiliates, public organizations, structural funders)
```

* boosting awareness of public code
* setting standards and creating reusable resources (by providing thought leadership!)
* demonstrating the value of public code with well documented, public business cases and risk analyses
* nurturing our community online, and – where sensible – in person

> ## Calls to action

Reviewer: Not sure whether the content that follows are calls to action. When I see "call to action", I expect some kind of arguments that will make me want to do something e.g. Join us in the fight against climate change! 

When I read the content that follows below, I see different types of information that we communicate about to different groups/stakeholders. Could we come up with a heading that's more fitting?

Author: On a website, a call to action is [any action we invite the user to take](https://www.optimizely.com/nl/optimization-glossary/call-to-action/). A call to action shouldn't be persuasive; instead, it should help or prompt the user to accomplish a goal they have already formulated themselves (especially while they've read the preceding content).

### Decision makers and sponsors

This includes:

* [political](../value-and-impact/user-mapping/political.html)
* [senior civil servants](../value-and-impact/user-mapping/senior-civil-servants.html)
* [structural funding and philanthropy](../value-and-impact/user-mapping/structural-funding-philanthropy.html)

Messages:

* Find out more about [place or institution’s] interesting public code(ish) approach/project
* Find out more about the Foundation for Public Code and its work (new member signed up, new codebase incubated)

> * Find out more about the Foundation for Public Code its vision

Reviewer: ```suggestion
* Find out more about the Foundation for Public Code and its vision
```

* Find out more about the Standard and our other resources for different user groups
* Come to meetups and events
* News from and profiles of our contributor community (‘join our thriving community!’)

### Other interested parties

This includes:

* vendors and the private sector
* end users (residents and citizens)
* journalists, academia and non-governmental organizations

Messages:

* Good news from our partners and affiliates (eg, relevant session at organization X’s annual conference!)
* Any news about Foundation or codebase impact for end users (eg, check out these communications by a member about their new software for managing parking)
* State of the market info about/for vendors (focusing on a class of vendors, rather than endorsing specific vendors)
* Potentially: announcing when a new vendor becomes available to support a specific codebase (like [this OS2 example](https://os2.eu/blog/en-ny-leverandoer-til-os2kitos-fra-juni-2019))

## Channels

> Our channels are:

Reviewer: Should we consider adding events as a channel? They are mentioned later on as things we do campaigns on, but are not explained anywhere here.

This would make our channels of communication more diverse, and not just restricted to social media platforms + blog. 

A thought for later: when we (re)establish our press contacts, would these also be worth mentioning here?

Author: > [This comment has a soundtrack.](https://www.youtube.com/watch?v=WySzEXKUSZw)

I don't consider events a communications channel. The other channels are persistent, free to access, can be indexed by search engines and found well into the future, and all contain (interlinked or cumulative) content that we create for others to consume privately and asynchonously. Though we often communicate at events and host our own, and craft original content for them, most of the magic comes from being in the room with the other participants. 
It's best practice for us to share our event content publicly afterwards so it's accessible to people who couldn't make the event, but we do that  via the other communcations channels.

* [Our blog](https://blog.publiccode.net/): long form, platform of record (read more about our blog)
* [Twitter](https://twitter.com/publiccodenet/): short form, ephemera

> * LinkedIn: share our blogposts or reshare others' content about us, ephemera (mostly avoid, since it’s a closed platform)

Reviewer: ```suggestion
* [LinkedIn](https://www.linkedin.com/company/publiccodenet/): share our blogposts or re-share others' content about us, ephemera (mostly avoid, since it’s a closed platform)
```

* Facebook: medium form, ephemera (mostly avoid, since it’s a closed platform)

Our posting frequency is:

> * Twitter - as often as we have something to say, up to daily. It’s dynamic, responsive, topical

Reviewer: ```suggestion
* Twitter: as often as we have something to say, up to daily. It’s dynamic, responsive, topical
```

Our communications are in plain English, but we link to highly relevant pages in other languages, for example about our members’ work.

Our communications should show to showcase the Foundation as:

* active, dynamic, getting things done
* smart enough to be trusted with the importance and complexity of its work
* plugged into and supportive of the network of visionaries and activists who also want this future
* open, both about its intentions and operations (transparency) as well as learnings, failures and successes

We believe it’s more important to talk concretely about what we’ve already done than what we hope to do.

## Mission statement

To enable public-purpose software and policy that is open and collaborative.

## Business objectives

1. Get more members ([in phase 1](https://about.publiccode.net/organization/roadmap.html))
2. Boost public codebases under Foundation stewardship
3. Build our codebase community and attract more contributors
4. Get more contributors and funding for the Foundation
5. Boost the public code ecosystem

## Audience

Our audience is divided into:

* current Foundation users

> * future Foundation users

Reviewer 2: Even though it has some content below, I think 'user' really needs some sort of definition here, as it feels quite out of place.

Author: This use is defo out of sync with #603.

* influencers

### Foundation users

We believe [these are the users of the Foundation for Public Code](https://about.publiccode.net/activities/value-and-impact/user-mapping/).

> 

Reviewer 2: ```suggestion
```

* has a reproducible process suitable not just for cities, but all public organizations (unlike SCORE)
* is international (unlike OS2, Cloud 4 Cities, VNG-Realisatie, Adullact)

We are also actively building our ecosystem by:

* connecting members with like-minded parties (potential members, partners, affiliates, structural funders)
* boosting awareness of public code
* setting standards and creating reusable resources (by providing thought leadership!)
* demonstrating the value of public code with well documented, public business cases and risk analyses
* nurturing our community online, and at conferences and events

## Calls to action

### Decision makers and sponsors

This includes:

* [political](../value-and-impact/user-mapping/political.html)
* [senior civil servants](../value-and-impact/user-mapping/senior-civil-servants.html)
* [structural funding and philanthropy](../value-and-impact/user-mapping/structural-funding-philanthropy.html)

Messages:

* Find out more about [place or institution’s] interesting public code(ish) approach/project
* Find out more about the Foundation’s work (new member signed up, new codebase incubated)

> * Find out more about the Foundation’s vision

Reviewer 2: ```suggestion
* Find out more about the Foundation for Public Code its work (new member signed up, new codebase incubated)
* Find out more about the Foundation for Public Code its vision
```

* Find out more about the Standard and our other resources for different user groups
* Come to meetups and events
* News from and profiles of our contributor community (‘join our thriving community!’)

### Other interested parties

This includes:

* vendors and the private sector
* end users (residents and citizens)
* journalists, academia and non-governmental organizations

Messages:

* Good news from our partners and affiliates (eg, relevant session at organization X’s annual conference!)
* Any news about Foundation or codebase impact for end users (eg, check out these communications by a member about their new software for managing parking)
* State of the market info about/for vendors (focusing on a class of vendors, rather than endorsing specific vendors)
* Potentially: announcing when a new vendor becomes available to support a specific codebase (like [this OS2 example](https://os2.eu/blog/en-ny-leverandoer-til-os2kitos-fra-juni-2019))

## Channels

Our channels are:

* [Our blog]: long form, platform of record (read more about our blog)
* [Twitter]: short form, ephemera
* [Facebook]: medium form, ephemera (mostly avoid, since it’s a closed platform)

> * [LinkedIn]: reshare content created about us and posted by others, or replicate FB (mostly avoid, since it’s a closed platform)

Reviewer 2: These should be links I'm guessing. Except for Facebook and LinkedIn that should just be words.

Our messages are positive and inspirational. Our community won’t be sustainable if we attract people motivated by fear or anger.

Our communications are in plain English, but we link to highly relevant pages in other languages, for example about our members’ work.

Our communications should show to showcase the Foundation as:

- active, dynamic, getting things done
- smart enough to be trusted with the importance and complexity of its work
- plugged into and supportive of the network of visionaries and activists who also want this future
- open, both about its intentions and operations (transparency) as well as learnings, failures and successes

We believe it’s more important to talk concretely about what we’ve already done than what we hope to do.

## Mission statement

To enable public-purpose software and policy that is open and collaborative.

## Business objectives

1. Get more members ([in phase 1](https://about.publiccode.net/organization/roadmap.html))
2. Boost public codebases under Foundation stewardship
3. Build our codebase community and attract more contributors
4. Get more contributors and funding for the Foundation

> 5. Boost the public code ecosystem

Author: I've replaced this with a link to the soon-to-be-published priorities.

Scheduling days of the week for maximum engagement

- We aim to publish blogposts and jobs on Mondays (for maximum engagement).
- The bulk of our social media engagement should take place Monday to Thursday.

## Messaging matrix

We don’t know what the ideal ratio of audiences to target is yet. We’ll develop as we learn more about who actually engages with us.

## Campaigns

We run a campaign every time:

- someone from the Foundation speaks at an event
- we hit a [milestone](https://github.com/publiccodenet/about/pull/224)

### Campaign template

This should be tied into general strategy sessions, so we get the maximum impact from each event or milestone. These are generally structured as:

- Announcement that thing is happening (give teaser)
- Join in on other related conversations and hashtags (monitor conversation)
- Tweet about the thing as it’s happening
- Publish blog during (if a major announcement/unveiling) or afterwards about what happened, plus what we learned or what’s next
- Measure how this worked

## Metrics

> These are statistics we can easily track month on month as a proxy for impact:

Author: I've removed the whole metrics section, and instead created #646.

* tweet numbers
* new Twitter followers
* Twitter profile visits
* Twitter mentions
* tweet impressions

> * increased use of #publiccode

Author: I've removed the whole metrics section.
